### DISCUSSION TITLE
Unable to create kv database

### DISCUSSION THREAD

**Discussion by joshholly** [vercel/community#4063] [Bug Report]
### Page to Investigate

https://vercel.com/dashboard/vercel/stores

### Steps to Reproduce

When I go to https://vercel.com/dashboard/vercel/stores , or when I click on the Get Started on the Storage plans for Vercel KV, it says Error 404. 

<img width="1624" alt="Screenshot 2023-09-13 at 9 04 21 PM" src="https://github.com/vercel/community/assets/1297910/0ce91c84-7474-4f1c-abcc-25671bcabc81">

**Comment by amyegan** [comment]
Hey @joshholly. Thanks for reporting this. 

I didn't run into any errors when testing it just now. Is this still a problem for you?

**Reply by sohanemon** [reply]
Having the same issue still.
